src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] colors = {"blue", "red", "green", "yellow", "black"};
+
+    public String getRandomColor() {
+        return colors[new Random().nextInt(colors.length)];
```

Comment:
random should be private final field
```suggestion
        return colors[random.nextInt(colors.length)];
```